app/javascript/src/component_dialog_form.js

```diff
@@ -158,16 +150,10 @@ class DialogForm {
   #build() {
     var dialog = this;
     
-    if(this.#config.activator) {
+    if(this.activator) {
```

Comment:
Yeah, this is because activator can be one of three values:
1) false (default) - i.e. I don't want one.
2) true - I want one, make it for me
3) $node - I want an activator, but I already have it - here it is.

So if `this.activator` is true we call `#createActivator` which calls `new DialogActivator` which first checks if it's been given a jQuery node.  If it has then it does nothing.

Maybe not the most intuitive, but I think it sort of arose from what was already there... will think on possible improvement